### ISSUE TITLE
Feature: wallet:export and wallet:address changes

### ISSUE THREAD

**Issue by irontankfish** [iron-fish/ironfish#3615]
### Description

I noticed that in the discord chat, people were confused where they could get their public key, and what the export address was. In fact, it is a bit confusing for someone who isn't a developer.

There are two potential fixes. You can decide which one to pick:
1. Ironfish wallet:export
Private key: xyz
Keep this address in a secure place. You can use it later to recover your wallet

Ironfish wallet:address
Public key: xyz
You can use this address for sending and receiving IRON.

2. Ironfish wallet:privateKey
Private key: xyz
Keep this address in a secure place. You may use it later to import your wallet

Ironfish wallet:publicKey
Public key: xyz
You may use this address for sending and receiving IRON.

This is a quick fix, so I can get started on fixing it.

**Comment by wd021** [comment]
i think a lot of the confusion was people from previous versions who weren't aware of the changes made recently.

**Comment by irontankfish** [comment]
Yes, I agree. But adding in the message after displaying the private, and public key would be useful for current and future users, which I suggest in the feature request. Currently we don't have any message displaying, which I feel is important when displaying the keys to our users.

We can also change the name of export, and address to publicKey, and privateKey, but this would be up to the developers, whether they might prefer it that way or not.
